examples/anime-http/server/asyncapi.yaml

```diff
@@ -0,0 +1,114 @@
+servers:
+  trendingAnimeServer:
+    url: http://localhost:8081
+    protocol: http
+channels:
+  trendingAnime:
+    bindings:
+      http:
+        type: request
+        method: POST
+        bindingVersion: 0.1.0
+        query:
+          type: object
+          required:
+            - name
+            - rating
+            - genre
+            - studio
+          properties:
+            name:
+              type: string
+              description: name of the anime.
+            rating:
+              type: string
+              description: rating of the show.
+            genre:
+              type: string
+              description: The genre of anime.
+            studio:
+              type: string
+              description: The studio of anime.
+        body:
+          type: object
+          required:
+            - name
+            - rating
+            - genre
+            - studio
+          properties:
+            name:
+              type: string
+              description: name of the anime.
+            rating:
+              type: number
+              description: rating of the show.
+            genre:
+              type: string
+              description: The genre of anime.
+            studio:
+              type: string
+              description: The studio of anime.
+    publish:
+      operationId: trendingAnimeController
+      message:
+        $ref: "#/components/messages/trendingAnime"
+    subscribe:
+      message:
+        payload:
+          type: object
+components:
+  messages:
+    trendingAnime:
+      summary: Data required to populate trending anime
+      payload:
+        type: object
+        required:
+          - query
+          - body
+        properties:
+          query:
+            type: object
+            required:
+              - name
+              - rating
+              - genre
+              - studio
+            properties:
+              name:
+                type: string
+                description: name of the anime.
+              rating:
+                type: string
+                description: rating of the show.
+              genre:
+                type: string
+                description: The genre of anime.
+              studio:
+                type: string
+                description: The studio of anime.
+          body:
+            type: object
+            required:
+              - name
+              - rating
+              - genre
+              - studio
+            properties:
+              name:
+                type: string
+                description: name of the anime.
+              rating:
+                type: number
+                description: rating of the show.
+              genre:
+                type: string
+                description: The genre of anime.
+              studio:
+                type: string
+                description: The studio of anime.
```

Comment:
This is almost the same as above. Let's use `components/schemas` to avoid repetition.